docs/source/en/tutorials/inference_with_lcm_lora.md

```diff
@@ -0,0 +1,406 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Performing inference with LCM-LoRA
```

Comment:
```suggestion
# Latent Consistency Model-LoRA
```

Reply:
@stevhliu Community is more aware of LCM-LoRA. I think it might be better to keep the original title as is both for respecting the awareness and for SEO (I don't have a strong affinity for this point, but recently, it's becoming more and more relevant).